src/components/organisms/Navbar/Navbar/Navbar.tsx

```diff
@@ -295,7 +295,7 @@ const NavbarWrapper: React.FC<NavbarProps> = ({
             <LayoutInner>
               {links ? (
                 <HamburgerContainer open={open} onClick={() => setOpen(!open)} data-testid="hamburger-icon">
-                  <Icon variant={faBars} color={open ? colors.brand : colors.white} fixedWidth />
+                  <Icon variant={faBars} color={open ? colors.brand : colors.white} width="24px" height="24px" />
```

Comment:
hamburger menu should be 20px. Rob requested logo, hamburger and profile icon all same height

Reply:
will do!